modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -373,6 +373,10 @@ definition {
 	test CanOrderByCreatedBy {
 		var siteName = TestCase.getSiteName(siteName = ${siteName});
 
+		Site.openSiteSettingsAdmin(siteURLKey = "guest");
```

Comment:
Added these 2 macros to tests that include membership. Membership needs to be enabled through site settings for it to be visible in the side panel.

Reply:
The change looks fine to me but I'll let Patty take a look once she's back